applications/nrf_desktop/README.rst

```diff
@@ -788,6 +788,12 @@ After building the application with or without :ref:`specifying the build type <
 
 After the connection is established and the device is bonded, you can use the mouse with the connected device.
 
+Windows HLK tests
+-----------------
+
+The nRF Desktop devices are confirmed to pass the tests from official playlist (:file:`HLK Version 1903 CompatPlaylist x86 x64 ARM64.xml`) required for compatibility with Windows 10 by Windows Hardware Compatibility Program.
+The tests were conducted using Windows Hardware Lab Kit.
+
 ----
```

Comment:
```suggestion

```
Please remove the divider. It is now automatically added to *** headings. (Check for other dividers in the file, please.)

Reply:
Ok